app/scripts/metamask-controller.js

```diff
@@ -511,7 +513,7 @@ export default class MetamaskController extends EventEmitter {
         // that is what the handler name implies, but this triggers too soon
         // causing the provider of the AssetsContractController to trail the
         // network provider by one update.
```

Comment:
this comment block is now stale and should probably be removed

Reply:
🙏